README.md

```diff
@@ -28,6 +28,7 @@ Following are the guideliens to author new Azure Actions and also to contribute
   - [Azure Actions toolkit](https://github.com/Azure/actions-toolkit) hosting node modules to help write your azure actions
   - [Guidelines for repository permissions](docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#guidelines-for-setting-permissions-on-the-repo)
   - [Action Versioning](docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#action-versioning)
+  - [Releasing a new version](docs/release-process.md)
   - [Testing GitHub Actions](docs/Testing-GitHub-Actions.md)

```

Comment:
@BALAGA-GAYATRI , for the test automation guideline also we should add a doc here with the approaches you have in place.

Reply:
@BALAGA-GAYATRI can you please take care of this in a separate PR? Will be merging this PR to have a proper link to the release process.